apps/client/src/dashboard/dashboard.page-crumbs.ts

```diff
@@ -0,0 +1,7 @@
+import type { PageCrumb } from '../breadcrumbs/page-crumb.type';
+
+export const DASHBOARD_PAGE_CRUMBS: PageCrumb[] = [
+  { text: 'Dashboards', href: '/dashboards' },
+  // TODO: Replace with real dashboard name once resource is being loaded
+  { text: 'Dashboard Name', href: 'no-op' }, // no link needed for deepest resource
```

Comment:
Won't this take us to no-op and show 404 since we don't have a route for this? Can we just leave this empty?

Reply:
There's no link rendered by Cloudscape for the deepest breadcrumb. I added a unit test to validate the expectation. The field is required no matter the depth. It could be replaced with an empty string, but there wouldn't be any behavioral difference.